css/nav.css

```diff
@@ -32,10 +32,7 @@ nav a:hover{
 .logo-name{
      color:#f84258;
 }
-.logo-name:hover{
```

Comment:
Removing this thing will affect the logo name and on hover you'll see 'Merlin Fashion' in color white. I suppose it's better to not change anything here.